sdk/storage/storage-file-share/src/ShareDirectoryClient.ts

```diff
@@ -1251,11 +1294,14 @@ export class ShareDirectoryClient extends StorageClient {
     );
     try {
       marker = marker === "" ? undefined : marker;
-      return await this.context.forceCloseHandles("*", {
+      const rawResponse = await this.context.forceCloseHandles("*", {
         marker,
         ...options,
         spanOptions
       });
+      const response = rawResponse as DirectoryForceCloseHandlesResponse;
+      response.closedHandlesCount = rawResponse.numberOfHandlesClosed || 0;
```

Comment:
minor: closedHandlesCount is good to me, while what's the reason to aliasing it? It looks the name is not aligned across languages.

Reply:
One reason is to avoid conflict with original optional property `numberOfHandlesClosed` while `closedHandlesCount` is required.